examples/microarch.yml

```diff
@@ -0,0 +1,32 @@
+type: fx32.32
+ioSync: Sync
+networks:
+  - name: net1
+    pus:
+      - type: SPI
+        name: spi
+        mosi: mosi
+        miso: miso
+        sclk: sclk
+        cs: cs
+        isSlave: true
+        bufferSize: 6
+        bounceFilter: 0  
+    protos:
+      - type: Fram
+        name: fram{x} # If you want a PU can be allocated only once, remove {x} from the PU name.
+        size: 32
+      - type: Shift
+        name: shift{x}
+        sRight: true
+      - type: Multiplier
+        name: mul{x}
+        mock: true
+      - type: Accum
+        name: accum{x}
+        isInt: true
+      - type: Divider
+        name: div{x}
+        mock: true
```

Comment:
Mock -- attr used for logic simulation via icarus verilog for complex PU, like divider.

Usually, we don't need to specify it for PUs independently, so it will be nice to move it on the top level.